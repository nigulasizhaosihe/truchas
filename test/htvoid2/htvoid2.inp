HTVOID2 -- Heat transfer in flow with void, Test 2

This exercises the treatment of void within the diffusion solver's
heat transfer model.  The focus here is on the interaction of boundary
conditions with the dynamic domain of the heat equation.  The domain is
a 1D flow channel with a 4x4 cell cross section, with wall bodies (1 cell
thick) on the top and bottom of the channel.  There is a plug of fluid
near the left end, with void on both sides of the plug.  The plug is
initially at rest but accelerates down the channel due to the gravity
which is directed to the right.  The simulation terminates before the
fluid has reached the right end.  Unlike HTVOID1, this mesh is orthogonal.
The wall bodies have a high heat capacity relative to the fluid. The interface
between the fluid and wall bodies are modeled with an internal HTC condition.
The back side of the channel is modeled with an external HTC condition
and a no-heat-flux condition is imposed on the front side.  The fluid
is initial at a uniform temperature, but develops a distinctive profile
as it propagates down the channel due to the boundary and interface
conditions.

The flow itself is extremely sensitive to the various flow parameters
(but completely independent of the heat transfer).  Small changes to
things -- initial step size, for example -- will lead to different
flow solutions with bits of fluid left behind or not.  The flow
solution is largely correct, but the differences in which cells are
void will result in heat solutions that are not comparable.

&MESH
  mesh_file = 'channel+mold-4.gen'
  interface_side_sets = 5, 6
/

&OUTPUTS
  Output_T  = 0.0, 1.0
  Output_Dt = 1.0 !.05
/

&PHYSICS
  materials = 'wall', 'fluid', 'VOID'
  flow = .true.
  body_force_density = 6.0, 0.0, 0.0
  heat_transport = .true.
/

&DIFFUSION_SOLVER
  stepping_method    = 'Non-adaptive BDF1'
  cond_vfrac_threshold = 1.0e-3
  residual_rtol      = 1.0e-6
  max_nlk_itr        = 10
  nlk_preconditioner = 'hypre_amg'
  verbose_stepping   = .true.
/

&NUMERICS
  dt_init = 0.01
  dt_grow = 1.05
/

&FLOW
  inviscid = .true.
  courant_number = 0.4
  vol_track_subcycles = 2
/

&FLOW_PRESSURE_SOLVER
  rel_tol = 0.0
  abs_tol = 1e-12
  max_ds_iter = 50
  max_amg_iter = 10
  krylov_method = "cg"
/

FLUID PLUG
&BODY
  surface_name    = 'from mesh file'
  mesh_material_number = 1
  material_name = 'fluid'
  velocity        = 3*0.0
  temperature     = 1.0
/

REMAINING VOID IN CHANNEL
&BODY
  surface_name = 'from mesh file'
  mesh_material_number = 2
  material_name = 'VOID'
  temperature = 0.0
/

LOWER WALL
&BODY
  surface_name = 'from mesh file'
  mesh_material_number = 3
  material_name = 'wall'
  temperature_function = 'init-temp-lower'
/

T = 2 - 0.25*x
&FUNCTION
  name = 'init-temp-lower'
  type = 'polynomial'
  poly_coefficients   = 2.0, -0.25
  poly_exponents(1,:) = 0, 1
/

UPPER WALL
&BODY
  surface_name = 'from mesh file'
  mesh_material_number = 4
  material_name = 'wall'
  temperature_function = 'init-temp-upper'
/

T = 1 + 0.125*x
&FUNCTION
  name = 'init-temp-upper'
  type = 'polynomial'
  poly_coefficients   = 1.0, 0.125
  poly_exponents(1,:) = 0, 1
/

&THERMAL_BC
  name = 'channel ends, front side'
  face_set_ids = 1, 2, 3
  type = 'flux'
  flux = 0.0
/

&THERMAL_BC
  name = 'back-side external HTC'
  face_set_ids = 4
  type = 'htc'
  htc = 0.2
  ambient_temp = 0.0
/

&THERMAL_BC
  name = 'wall bodies'
  face_set_ids = 7
  type = 'flux'
  flux = 0.0
/

&THERMAL_BC
  name = 'lower channel-wall interface'
  face_set_ids = 5
  type = 'interface-htc'
  htc = 20.0
/

&THERMAL_BC
  name = 'upper channel-wall interface'
  face_set_ids = 6
  type = 'interface-htc'
  htc = 0.2
/

&MATERIAL
  name = 'wall'
  density = 10.0
  specific_heat = 1.0
  conductivity = 1.0
/

&MATERIAL
  name = 'fluid'
  is_fluid = T
  density = 1.0
  specific_heat = 1.0
  conductivity = 2.0e-2
/
