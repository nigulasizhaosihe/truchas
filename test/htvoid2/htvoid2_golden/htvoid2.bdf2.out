HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.608E-01 (initial)
  HC error: ||F||_2 = 3.627E-05, scaled = 1.005E+02
    1: error= 1.00510E+02
  HC error: ||F||_2 = 9.481E-07, scaled = 2.628E+00
    2: error= 2.62750E+00
  HC error: ||F||_2 = 3.209E-10, scaled = 8.895E-04
    3: error= 8.89465E-04
  NLK BCE solve succeeded:   3 iterations, error= 8.89465E-04
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.796E-01 (initial)
  HC error: ||F||_2 = 3.671E-05, scaled = 9.669E+01
    1: error= 9.66933E+01
  HC error: ||F||_2 = 9.570E-07, scaled = 2.521E+00
    2: error= 2.52095E+00
  HC error: ||F||_2 = 1.677E-09, scaled = 4.417E-03
    3: error= 4.41730E-03
  NLK BCE solve succeeded:   3 iterations, error= 4.41730E-03
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.881E-05 (initial)
  HC error: ||F||_2 = 4.468E-08, scaled = 2.375E+03
    1: error= 2.37544E+03
  HC error: ||F||_2 = 1.130E-09, scaled = 6.008E+01
    2: error= 6.00840E+01
  HC error: ||F||_2 = 1.391E-11, scaled = 7.395E-01
    3: error= 7.39530E-01
  NLK BCE solve succeeded:   3 iterations, error= 7.39530E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.881E-05 (initial)
  HC error: ||F||_2 = 4.954E-08, scaled = 2.634E+03
    1: error= 2.63414E+03
  HC error: ||F||_2 = 1.247E-09, scaled = 6.631E+01
    2: error= 6.63075E+01
  HC error: ||F||_2 = 4.442E-13, scaled = 2.361E-02
    3: error= 2.36146E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.36146E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.027E-05 (initial)
  HC error: ||F||_2 = 5.501E-08, scaled = 2.714E+03
    1: error= 2.71404E+03
  HC error: ||F||_2 = 1.381E-09, scaled = 6.813E+01
    2: error= 6.81339E+01
  HC error: ||F||_2 = 4.993E-13, scaled = 2.463E-02
    3: error= 2.46339E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.46339E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.194E-05 (initial)
  HC error: ||F||_2 = 6.106E-08, scaled = 2.783E+03
    1: error= 2.78333E+03
  HC error: ||F||_2 = 1.528E-09, scaled = 6.966E+01
    2: error= 6.96584E+01
  HC error: ||F||_2 = 5.650E-13, scaled = 2.575E-02
    3: error= 2.57549E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.57549E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.384E-05 (initial)
  HC error: ||F||_2 = 6.778E-08, scaled = 2.844E+03
    1: error= 2.84362E+03
  HC error: ||F||_2 = 1.690E-09, scaled = 7.092E+01
    2: error= 7.09171E+01
  HC error: ||F||_2 = 6.436E-13, scaled = 2.700E-02
    3: error= 2.70020E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.70020E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.599E-05 (initial)
  HC error: ||F||_2 = 7.526E-08, scaled = 2.896E+03
    1: error= 2.89610E+03
  HC error: ||F||_2 = 1.869E-09, scaled = 7.194E+01
    2: error= 7.19365E+01
  HC error: ||F||_2 = 7.386E-13, scaled = 2.842E-02
    3: error= 2.84231E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.84231E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.840E-05 (initial)
  HC error: ||F||_2 = 8.357E-08, scaled = 2.942E+03
    1: error= 2.94215E+03
  HC error: ||F||_2 = 2.066E-09, scaled = 7.275E+01
    2: error= 7.27535E+01
  HC error: ||F||_2 = 8.544E-13, scaled = 3.008E-02
    3: error= 3.00810E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.00810E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.111E-05 (initial)
  HC error: ||F||_2 = 9.281E-08, scaled = 2.983E+03
    1: error= 2.98327E+03
  HC error: ||F||_2 = 2.284E-09, scaled = 7.342E+01
    2: error= 7.34202E+01
  HC error: ||F||_2 = 9.974E-13, scaled = 3.206E-02
    3: error= 3.20595E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.20595E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.413E-05 (initial)
  HC error: ||F||_2 = 1.031E-07, scaled = 3.021E+03
    1: error= 3.02095E+03
  HC error: ||F||_2 = 2.526E-09, scaled = 7.401E+01
    2: error= 7.40130E+01
  HC error: ||F||_2 = 1.177E-12, scaled = 3.447E-02
    3: error= 3.44723E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.44723E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.749E-05 (initial)
  HC error: ||F||_2 = 1.144E-07, scaled = 3.052E+03
    1: error= 3.05233E+03
  HC error: ||F||_2 = 2.790E-09, scaled = 7.443E+01
    2: error= 7.44265E+01
  HC error: ||F||_2 = 1.436E-12, scaled = 3.830E-02
    3: error= 3.83026E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.83026E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.122E-05 (initial)
  HC error: ||F||_2 = 1.271E-07, scaled = 3.084E+03
    1: error= 3.08353E+03
  HC error: ||F||_2 = 3.017E-09, scaled = 7.320E+01
    2: error= 7.31974E+01
  HC error: ||F||_2 = 6.471E-12, scaled = 1.570E-01
    3: error= 1.57011E-01
  NLK BCE solve succeeded:   3 iterations, error= 1.57011E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.534E-05 (initial)
  HC error: ||F||_2 = 1.414E-07, scaled = 3.119E+03
    1: error= 3.11874E+03
  HC error: ||F||_2 = 3.359E-09, scaled = 7.408E+01
    2: error= 7.40764E+01
  HC error: ||F||_2 = 8.257E-12, scaled = 1.821E-01
    3: error= 1.82109E-01
  NLK BCE solve succeeded:   3 iterations, error= 1.82109E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.991E-05 (initial)
  HC error: ||F||_2 = 1.575E-07, scaled = 3.156E+03
    1: error= 3.15632E+03
  HC error: ||F||_2 = 3.756E-09, scaled = 7.525E+01
    2: error= 7.52520E+01
  HC error: ||F||_2 = 1.083E-11, scaled = 2.171E-01
    3: error= 2.17086E-01
  NLK BCE solve succeeded:   3 iterations, error= 2.17086E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 5.497E-05 (initial)
  HC error: ||F||_2 = 1.758E-07, scaled = 3.198E+03
    1: error= 3.19802E+03
  HC error: ||F||_2 = 4.213E-09, scaled = 7.663E+01
    2: error= 7.66333E+01
  HC error: ||F||_2 = 1.474E-11, scaled = 2.682E-01
    3: error= 2.68174E-01
  NLK BCE solve succeeded:   3 iterations, error= 2.68174E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 6.061E-05 (initial)
  HC error: ||F||_2 = 1.968E-07, scaled = 3.247E+03
    1: error= 3.24667E+03
  HC error: ||F||_2 = 4.739E-09, scaled = 7.818E+01
    2: error= 7.81788E+01
  HC error: ||F||_2 = 2.023E-11, scaled = 3.338E-01
    3: error= 3.33755E-01
  NLK BCE solve succeeded:   3 iterations, error= 3.33755E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 6.696E-05 (initial)
  HC error: ||F||_2 = 2.213E-07, scaled = 3.306E+03
    1: error= 3.30585E+03
  HC error: ||F||_2 = 5.348E-09, scaled = 7.988E+01
    2: error= 7.98755E+01
  HC error: ||F||_2 = 2.758E-11, scaled = 4.119E-01
    3: error= 4.11921E-01
  NLK BCE solve succeeded:   3 iterations, error= 4.11921E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 6.111E-02 (initial)
  HC error: ||F||_2 = 7.557E-07, scaled = 1.237E+01
    1: error= 1.23671E+01
  HC error: ||F||_2 = 1.839E-08, scaled = 3.009E-01
    2: error= 3.00903E-01
  NLK BCE solve succeeded:   2 iterations, error= 3.00903E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.890E-04 (initial)
  HC error: ||F||_2 = 1.721E-06, scaled = 9.106E+03
    1: error= 9.10592E+03
  HC error: ||F||_2 = 4.212E-08, scaled = 2.229E+02
    2: error= 2.22882E+02
  HC error: ||F||_2 = 1.861E-11, scaled = 9.846E-02
    3: error= 9.84599E-02
  NLK BCE solve succeeded:   3 iterations, error= 9.84599E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.058E-04 (initial)
  HC error: ||F||_2 = 3.093E-07, scaled = 2.924E+03
    1: error= 2.92434E+03
  HC error: ||F||_2 = 7.372E-09, scaled = 6.970E+01
    2: error= 6.96964E+01
  HC error: ||F||_2 = 4.023E-11, scaled = 3.803E-01
    3: error= 3.80321E-01
  NLK BCE solve succeeded:   3 iterations, error= 3.80321E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.134E-04 (initial)
  HC error: ||F||_2 = 3.469E-07, scaled = 3.058E+03
    1: error= 3.05815E+03
  HC error: ||F||_2 = 8.270E-09, scaled = 7.290E+01
    2: error= 7.29013E+01
  HC error: ||F||_2 = 5.558E-11, scaled = 4.899E-01
    3: error= 4.89924E-01
  NLK BCE solve succeeded:   3 iterations, error= 4.89924E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.226E-04 (initial)
  HC error: ||F||_2 = 3.950E-07, scaled = 3.223E+03
    1: error= 3.22275E+03
  HC error: ||F||_2 = 9.433E-09, scaled = 7.696E+01
    2: error= 7.69602E+01
  HC error: ||F||_2 = 7.712E-11, scaled = 6.292E-01
    3: error= 6.29212E-01
  NLK BCE solve succeeded:   3 iterations, error= 6.29212E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 5.429E-02 (initial)
  HC error: ||F||_2 = 3.293E-06, scaled = 6.065E+01
    1: error= 6.06509E+01
  HC error: ||F||_2 = 7.917E-08, scaled = 1.458E+00
    2: error= 1.45828E+00
  HC error: ||F||_2 = 1.446E-09, scaled = 2.664E-02
    3: error= 2.66351E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.66351E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.978E-04 (initial)
  HC error: ||F||_2 = 9.316E-07, scaled = 1.871E+03
    1: error= 1.87145E+03
  HC error: ||F||_2 = 2.255E-08, scaled = 4.530E+01
    2: error= 4.52982E+01
  HC error: ||F||_2 = 1.131E-10, scaled = 2.273E-01
    3: error= 2.27278E-01
  NLK BCE solve succeeded:   3 iterations, error= 2.27278E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.711E-04 (initial)
  HC error: ||F||_2 = 5.619E-07, scaled = 3.285E+03
    1: error= 3.28483E+03
  HC error: ||F||_2 = 1.339E-08, scaled = 7.825E+01
    2: error= 7.82483E+01
  HC error: ||F||_2 = 1.320E-10, scaled = 7.715E-01
    3: error= 7.71469E-01
  NLK BCE solve succeeded:   3 iterations, error= 7.71469E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.835E-02 (initial)
  HC error: ||F||_2 = 4.560E-06, scaled = 9.432E+01
    1: error= 9.43151E+01
  HC error: ||F||_2 = 1.098E-07, scaled = 2.272E+00
    2: error= 2.27186E+00
  HC error: ||F||_2 = 1.561E-09, scaled = 3.229E-02
    3: error= 3.22944E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.22944E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 5.828E-04 (initial)
  HC error: ||F||_2 = 5.015E-07, scaled = 8.606E+02
    1: error= 8.60605E+02
  HC error: ||F||_2 = 1.207E-08, scaled = 2.072E+01
    2: error= 2.07195E+01
  HC error: ||F||_2 = 1.404E-10, scaled = 2.410E-01
    3: error= 2.40998E-01
  NLK BCE solve succeeded:   3 iterations, error= 2.40998E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.307E-02 (initial)
  HC error: ||F||_2 = 2.095E-06, scaled = 4.863E+01
    1: error= 4.86298E+01
  HC error: ||F||_2 = 5.047E-08, scaled = 1.172E+00
    2: error= 1.17174E+00
  HC error: ||F||_2 = 1.003E-09, scaled = 2.329E-02
    3: error= 2.32912E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.32912E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.488E-04 (initial)
  HC error: ||F||_2 = 2.281E-06, scaled = 6.540E+03
    1: error= 6.54041E+03
  HC error: ||F||_2 = 5.524E-08, scaled = 1.584E+02
    2: error= 1.58387E+02
  HC error: ||F||_2 = 1.779E-10, scaled = 5.100E-01
    3: error= 5.09968E-01
  NLK BCE solve succeeded:   3 iterations, error= 5.09968E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.604E-04 (initial)
  HC error: ||F||_2 = 3.913E-07, scaled = 2.440E+03
    1: error= 2.44017E+03
  HC error: ||F||_2 = 9.345E-09, scaled = 5.828E+01
    2: error= 5.82761E+01
  HC error: ||F||_2 = 1.334E-10, scaled = 8.319E-01
    3: error= 8.31897E-01
  NLK BCE solve succeeded:   3 iterations, error= 8.31897E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.762E-02 (initial)
  HC error: ||F||_2 = 3.942E-06, scaled = 1.048E+02
    1: error= 1.04780E+02
  HC error: ||F||_2 = 9.606E-08, scaled = 2.554E+00
    2: error= 2.55351E+00
  HC error: ||F||_2 = 1.272E-09, scaled = 3.381E-02
    3: error= 3.38067E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.38067E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 4.745E-04 (initial)
  HC error: ||F||_2 = 3.386E-07, scaled = 7.137E+02
    1: error= 7.13713E+02
  HC error: ||F||_2 = 8.179E-09, scaled = 1.724E+01
    2: error= 1.72382E+01
  HC error: ||F||_2 = 1.270E-10, scaled = 2.677E-01
    3: error= 2.67657E-01
  NLK BCE solve succeeded:   3 iterations, error= 2.67657E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.269E-02 (initial)
  HC error: ||F||_2 = 1.816E-06, scaled = 5.555E+01
    1: error= 5.55490E+01
  HC error: ||F||_2 = 4.451E-08, scaled = 1.362E+00
    2: error= 1.36160E+00
  HC error: ||F||_2 = 8.556E-10, scaled = 2.617E-02
    3: error= 2.61709E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.61709E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.918E-04 (initial)
  HC error: ||F||_2 = 1.945E-06, scaled = 6.666E+03
    1: error= 6.66593E+03
  HC error: ||F||_2 = 4.784E-08, scaled = 1.640E+02
    2: error= 1.63971E+02
  HC error: ||F||_2 = 1.988E-10, scaled = 6.815E-01
    3: error= 6.81508E-01
  NLK BCE solve succeeded:   3 iterations, error= 6.81508E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.452E-04 (initial)
  HC error: ||F||_2 = 2.877E-07, scaled = 1.981E+03
    1: error= 1.98104E+03
  HC error: ||F||_2 = 6.891E-09, scaled = 4.745E+01
    2: error= 4.74512E+01
  HC error: ||F||_2 = 1.194E-10, scaled = 8.224E-01
    3: error= 8.22444E-01
  NLK BCE solve succeeded:   3 iterations, error= 8.22444E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.766E-02 (initial)
  HC error: ||F||_2 = 3.365E-06, scaled = 1.216E+02
    1: error= 1.21639E+02
  HC error: ||F||_2 = 8.293E-08, scaled = 2.998E+00
    2: error= 2.99779E+00
  HC error: ||F||_2 = 9.991E-10, scaled = 3.612E-02
    3: error= 3.61160E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.61160E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 3.738E-04 (initial)
  HC error: ||F||_2 = 2.578E-07, scaled = 6.897E+02
    1: error= 6.89679E+02
  HC error: ||F||_2 = 6.127E-09, scaled = 1.639E+01
    2: error= 1.63916E+01
  HC error: ||F||_2 = 1.106E-10, scaled = 2.958E-01
    3: error= 2.95828E-01
  NLK BCE solve succeeded:   3 iterations, error= 2.95828E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.300E-02 (initial)
  HC error: ||F||_2 = 1.530E-06, scaled = 6.651E+01
    1: error= 6.65093E+01
  HC error: ||F||_2 = 3.752E-08, scaled = 1.631E+00
    2: error= 1.63122E+00
  HC error: ||F||_2 = 6.637E-10, scaled = 2.885E-02
    3: error= 2.88521E-02
  NLK BCE solve succeeded:   3 iterations, error= 2.88521E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.432E-04 (initial)
  HC error: ||F||_2 = 1.626E-06, scaled = 6.686E+03
    1: error= 6.68584E+03
  HC error: ||F||_2 = 4.001E-08, scaled = 1.645E+02
    2: error= 1.64519E+02
  HC error: ||F||_2 = 2.193E-10, scaled = 9.017E-01
    3: error= 9.01736E-01
  NLK BCE solve succeeded:   3 iterations, error= 9.01736E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.327E-04 (initial)
  HC error: ||F||_2 = 2.312E-07, scaled = 1.743E+03
    1: error= 1.74261E+03
  HC error: ||F||_2 = 5.377E-09, scaled = 4.053E+01
    2: error= 4.05298E+01
  HC error: ||F||_2 = 1.018E-10, scaled = 7.673E-01
    3: error= 7.67329E-01
  NLK BCE solve succeeded:   3 iterations, error= 7.67329E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.835E-02 (initial)
  HC error: ||F||_2 = 2.798E-06, scaled = 1.525E+02
    1: error= 1.52496E+02
  HC error: ||F||_2 = 6.885E-08, scaled = 3.752E+00
    2: error= 3.75188E+00
  HC error: ||F||_2 = 7.287E-10, scaled = 3.971E-02
    3: error= 3.97094E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.97094E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.844E-04 (initial)
  HC error: ||F||_2 = 2.149E-07, scaled = 7.557E+02
    1: error= 7.55670E+02
  HC error: ||F||_2 = 4.921E-09, scaled = 1.730E+01
    2: error= 1.73042E+01
  HC error: ||F||_2 = 9.638E-11, scaled = 3.389E-01
    3: error= 3.38900E-01
  NLK BCE solve succeeded:   3 iterations, error= 3.38900E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.405E-02 (initial)
  HC error: ||F||_2 = 1.248E-06, scaled = 8.877E+01
    1: error= 8.87688E+01
  HC error: ||F||_2 = 3.052E-08, scaled = 2.172E+00
    2: error= 2.17168E+00
  HC error: ||F||_2 = 4.506E-10, scaled = 3.206E-02
    3: error= 3.20572E-02
  NLK BCE solve succeeded:   3 iterations, error= 3.20572E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 2.012E-04 (initial)
  HC error: ||F||_2 = 1.329E-06, scaled = 6.607E+03
    1: error= 6.60713E+03
  HC error: ||F||_2 = 3.265E-08, scaled = 1.623E+02
    2: error= 1.62253E+02
  HC error: ||F||_2 = 2.351E-10, scaled = 1.169E+00
    3: error= 1.16857E+00
  HC error: ||F||_2 = 5.132E-13, scaled = 2.551E-03
    4: error= 2.55070E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.55070E-03
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.191E-04 (initial)
  HC error: ||F||_2 = 1.931E-07, scaled = 1.620E+03
    1: error= 1.62038E+03
  HC error: ||F||_2 = 4.199E-09, scaled = 3.524E+01
    2: error= 3.52422E+01
  HC error: ||F||_2 = 8.085E-11, scaled = 6.786E-01
    3: error= 6.78648E-01
  NLK BCE solve succeeded:   3 iterations, error= 6.78648E-01
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.010E-02 (initial)
  HC error: ||F||_2 = 2.059E-06, scaled = 2.039E+02
    1: error= 2.03919E+02
  HC error: ||F||_2 = 5.083E-08, scaled = 5.033E+00
    2: error= 5.03273E+00
  HC error: ||F||_2 = 5.029E-10, scaled = 4.980E-02
    3: error= 4.97975E-02
  NLK BCE solve succeeded:   3 iterations, error= 4.97975E-02
HEAT TRANSFER FUNCTION NORMS
  HC error: ||F||_2 = 1.949E-04 (initial)
  HC error: ||F||_2 = 2.863E-07, scaled = 1.469E+03
    1: error= 1.46910E+03
  HC error: ||F||_2 = 7.009E-09, scaled = 3.597E+01
    2: error= 3.59684E+01
  HC error: ||F||_2 = 1.217E-10, scaled = 6.246E-01
    3: error= 6.24552E-01
  NLK BCE solve succeeded:   3 iterations, error= 6.24552E-01
